packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/index.tsx

```diff
@@ -1,16 +1,22 @@
 import React from "react";
 
 import { Wizard } from "../common/Wizard";
+import { PublicodeProvider } from "../publicodes";
 import { initialState, stepReducer } from "./stepReducer";
 
-const SimulateurPreavisRetraite = ({ icon, title }) => (
-  <Wizard
-    icon={icon}
-    title={title}
-    initialState={initialState}
-    // @ts-ignore
-    stepReducer={stepReducer}
-  />
+const SimulateurPreavisRetraite = ({ icon, title, publicodes }) => (
+  <PublicodeProvider
+    publicodes={publicodes}
```

Comment:
Should we rename that to `rules` or `publicodesRules` or `rulesDefinitions`

Reply:
I think `rules` is a good idea. No need to use publicodes here because we are already in a `PublicodeProvider` and `rulesDefinitions` is fat.

I'll change it.